### DISCUSSION TITLE
When will CodeQL support dotnet7 and C#11?

### DISCUSSION THREAD

**Discussion by tarifrr** [github/codeql#11259] [General]
When will codeql support dotnet7 and C#11?

**Comment by michaelnebel** [comment]
The development will start in the first quarter next year: https://github.com/orgs/github/projects/4247/views/1?filterQuery=label%3Acodeql

**Reply by michaelnebel** [reply]
We expect to ship support for C#11 and .NET 7 in 2023Q2.

**Reply by tarifrr** [reply]
Thanks @michaelnebel . Appreciate the reply
